localstack/services/sns/sns_listener.py

```diff
@@ -127,10 +127,14 @@ def forward_request(self, method, path, data, headers):
 
             elif req_action == 'CreateTopic':
                 topic_arn = aws_stack.sns_topic_arn(req_data['Name'][0])
-                self._extract_tags(topic_arn, req_data)
+                tag_error_response = self._extract_tags(topic_arn, req_data, True)
```

Comment:
nit: I suggest that we rename `tag_error_response` to `tag_resource_success`, as later in line 132 we're returning an error if this value is *not* truthy.